portfolio_D.css

```diff
@@ -0,0 +1,408 @@
+@media only screen and (min-width: 33rem) {
```

Comment:
- [ ] The size for the desktop version should be between `992px`-`1100px`. please ensure you are using this range for the media query
- [ ] [OPTIONAL] You don't need to have a media query for the mobile version of the styling. it can simply be the default and the desktop version is added using the media query. This is what is known as a mobile-first design.